XLPagerTabStrip/XL/Controllers/XLTwitterPagerTabStripViewController.m

```diff
@@ -167,11 +177,6 @@ -(void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NS
     }
 }
 
--(void)dealloc
-{
-    [self.navigationView removeObserver:self forKeyPath:@"frame"];
```

Comment:
@ppaulojr Have you tried a @try wrapping this line? I'm gonna try to reproduce the issue with @HelOlhausen.